tests/test_scripts.py

```diff
@@ -139,11 +140,15 @@ def test_train_preference_comparisons_reward_norm(tmpdir, named_configs):
         config_updates=config_updates,
     )
     if "reward.normalize_output_running" in named_configs:
-        assert run.config["reward"]["net_kwargs"]["normalize_output_layer"] is not None
+        assert (
+            run.config["reward"]["net_kwargs"]["rew_normalize_class"] is not nn.Identity
+        )
     elif "reward.normalize_output_disable" in named_configs:
-        assert run.config["reward"]["net_kwargs"]["normalize_output_layer"] is None
+        assert run.config["reward"]["net_kwargs"]["rew_normalize_class"] is nn.Identity
     else:
-        assert run.config["reward"]["net_kwargs"]["normalize_output_layer"] is not None
+        assert (
+            run.config["reward"]["net_kwargs"]["rew_normalize_class"] is not nn.Identity
+        )
     assert run.status == "COMPLETED"
     assert isinstance(run.result, dict)
```

Comment:
You could consider checking the scale of the RL algorithm return reported in `run.result` to see if reward normalization is/isn't applied. Might not be consistent enough, but would be more of an integration test. (Right now that config could be completely ignored and this would still pass..)

Reply:
I think it would be better to test normalization somewhere else. `run.result` only records `reward_loss`, `reward_accuracy`, and non-wrapped `rollout` stats. This test aims to check the type of normalization applied is right under named_configs in `reward` ingredient.